drape_frontend/selection_shape_generator.cpp

```diff
@@ -228,15 +226,9 @@ drape_ptr<RenderNode> SelectionShapeGenerator::GenerateSelectionGeometry(ref_ptr
   {
     mapDataProvider.ReadFeatures([&points](FeatureType & ft)
     {
-      if (ft.GetGeomType() != feature::GeomType::Line || points.size() > 1)
-        return;
-      points.reserve(5);
-      ft.ForEachPoint([&points](m2::PointD const & pt)
-      {
-       if (points.empty() || !points.back().EqualDxDy(pt, mercator::kPointEqualityEps))
```

Comment:
Weren't duplicate points filtered here? Do we have such cases in the data?

Does it make sense to write a test for it?

Reply:
Features already have unique points.

Reply:
ACK